DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -223,7 +223,9 @@ function PreFirstEntry()
     foreach AllActors(class'#var(prefix)Lamp', lmp) {
         lmp.InitLight();
     }
```

Comment:
I wonder if this should be called when MenuChoice_AutoLamps is disabled, I don't think it should